src/traces/contour/attributes.js

```diff
@@ -44,13 +44,15 @@ module.exports = extendFlat({}, {
     },
     ncontours: {
         valType: 'integer',
-        dflt: 0,
+        dflt: 15,
+        min: 1,
         role: 'style',
         description: [
             'Sets the maximum number of contour levels. The actual number',
             'of contours will be chosen automatically to be less than or',
             'equal to the value of `ncontours`.',
-            'Has an effect only if `autocontour` is *true*.'
+            'Has an effect only if `autocontour` is *true* or if',
+            '`contours.size` is missing.'
```

Comment:
> Any reason not to allow this usage?

I can't think of one. Nice addition :+1: 

> Note that the converse is not allowed:

So,

```js
autocontour: true,
contours: {
  size: 10
}
```

ignores `contours.size`, but then how does

```js
autocontour: false,
contours: {
  size: 10
}
```

_pick_ `contours.start` and `contours.end` using `Axes.tickFirst`?